packages/atlas-service/src/main.spec.ts

```diff
@@ -13,6 +13,20 @@ function getListenerCount(emitter: EventEmitter) {
   }, 0);
 }
 
+const atlasAIServiceTests: {
+  functionName: 'getQueryFromUserInput' | 'getAggregationFromUserInput';
+  aiEndpoint: string;
+}[] = [
+  {
+    functionName: 'getQueryFromUserInput',
+    aiEndpoint: 'mql-query',
+  },
+  {
+    functionName: 'getAggregationFromUserInput',
+    aiEndpoint: 'mql-aggregation',
+  },
```

Comment:
If they are literally the same to the point that the tests are only switching the endpoint name mock, I wounder if it would make sense to implement them as such and not test the same thing copied over twice